### PR TITLE
Add JUnit test for FileHandler Class

### LINKED ISSUE #108: Bug: Exception due to invalid path
From my light testing, entering the `file path` with inverted commas seems to crash the program, see below:  
image  
Perhaps handle the exception, or parse the path from the input string, would be great 😅

### PR DESCRIPTION
**Pull request by ehz0ah**
Close #108 

- Added JUnit test for FileHandler Class
- Specifically, reading in and saving to external files

### CODE REVIEW

File: src/test/java/filehandler/FileHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class FileHandlerTest {
+    private FileHandler fileHandler;
+    private Parser parser;
+    private StudentList studentList;
+    private UI ui;
+    private ArrayList<Student> results;
+
+    private static final Path CSV_FILE_PATH = Paths.get("test.csv");
```
Comment (isaacsaw25) [inline src/test/java/filehandler/FileHandlerTest.java:32]: The CI tests are failing as `checkstyleMain` requires the static variable declaration first 😅

**Review by isaacsaw25 — CHANGES_REQUESTED** [review]
Put the static variables first, just a small change 😄

**Review by Holy-An — APPROVED** [review]
LGTM!

**Review by isaacsaw25 — APPROVED** [review]
Excellent comprehensiveness in your unit tests! 😄

**Pull request merged**